src_c/line.c

```diff
@@ -490,6 +490,85 @@ pg_line_flip_ip(pgLineObject *self, PyObject *_null)
     Py_RETURN_NONE;
 }
 
+double 
+_lerp_helper(float start, float end, float amount) {
+    return (1 - amount) * start + amount * end;
+}
```

Comment:
make this `PG_FORCE_INLINE`. I think you can save a multiplication if the calculation is `start + (end - start) * amount`.